source/Halibut/HalibutRuntime.cs

```diff
@@ -91,6 +94,7 @@ public HalibutRuntime(IServiceFactory serviceFactory, X509Certificate2 serverCer
             this.messageSerializer = messageSerializer;
             this.pollingReconnectRetryPolicy = pollingReconnectRetryPolicy;
             invoker = new ServiceInvoker(serviceFactory);
+            this.TimeoutsAndLimits = halibutTimeoutsAndLimits;
```

Comment:
If a sync Halibut runtime is created, would it make sense to enforce that `halibutTimeoutsAndLimits` should be null, so the caller knows these are not going to apply and we don't create confusion?

Reply:
Yes and done.